lib/compiler.js

```diff
@@ -60,4 +68,8 @@ const run = ({ config, emitJson, watchConfig }, log) => {
   }
 };
 
+function isWebpack5() {
```

Comment:
```suggestion
const isWebpack5 = () => {
```

And do please move this ahead of `run`